apps/sms/style/root.css

```diff
@@ -79,6 +79,31 @@ small {
   color: #999;
 }
 
+/* Tabs */
+form[role="dialog"][data-type="edit"] .bb-tablist:before {
+  bottom: 5.5rem ;
+}
+
+form[role="dialog"][data-type="edit"] .bb-tablist > li {
+  height: 5.5rem ;
+}
+
+.bb-tablist > li button {
+  background-position: 50% 50% ;
+}
+
+.bb-tablist > li:first-child button {
```

Comment:
Please try to avoid the first-child/nth-child/... pseudo class because it might have performance concern